test/integration/merkle-streamer/ll/claim/claim.t.sol

```diff
@@ -127,15 +126,21 @@ contract Claim_Integration_Test is MerkleStreamer_Integration_Test {
         givenIncludedInMerkleTree
         givenProtocolFeeZero
     {
+        test_Claim(ud(0));
```

Comment:
Let's use named parameters here:

```solidity
test_Claim({ protocolFee: ud(0) });
```

Reply:
`protocolFee` affects the deposit amount in the lockup linear stream. When `protocolFee` is non-zero, the amount deposit will be claim amount minus fee amount. So, now we have two test functions:

```solidity
    function test_Claim_ProtocolFeeNotZero() external {
        changePrank({ msgSender: users.admin });
        UD60x18 protocolFee = ud(0.1e18);
        comptroller.setProtocolFee({ asset: asset, newProtocolFee: protocolFee });

        test_Claim(protocolFee);
    }
```

and 

```solidity

    function test_Claim() givenProtocolFeeZero {
        test_Claim(ud(0));
    }
```

And to avoid DRY, I added an internal `test_Claim` function that takes `protocolFee` as input parameter.